examples/texture_atlas.rs

```diff
@@ -78,7 +79,7 @@ fn startup_system(
 fn setup_ground(mut commands: Commands) {
     commands.spawn(SpriteBundle {
         sprite: Sprite {
-            color: Color::DARK_GRAY,
+            color: DARK_GRAY.into(),
```

Comment:
As per the [migration guide](https://bevyengine.org/learn/migration-guides/0-13-to-0-14/#css-constants), `DARK_GRAY` should be now `Srgba::gray(0.25)` (please double check).

Reply:
I did not actually read the guide :joy:  - I'll change this.